README.rst

```diff
@@ -74,23 +74,37 @@ Settings
 g:coverage_script
     Default: "" (which means autodetect)
 
-    Name of the script that can produce reports.  Example ::
+    Name of the script that can produce reports. Example: ::
 
         let g:coverage_script = 'python3 -m coverage'
 
     By default it looks for ``coverage`` in your PATH, and if not found,
     it looks for ``bin/coverage`` relative to the current working
     directory.
 
+g:coverage_sign
+    Default: "↣"
+
+g:coverage_sign_branch
+    Default: "↦"
+
+g:coverage_sign_branch_target
+    Default: "⇥"
+
+    Signs used in sign_column (empty string allowed). Example: ::
+
+        let g:coverage_sign = '#'
+        let g:coverage_sign_branch_target = ''
+
 
 Highlight groups
 ----------------
 
-If you want to change the highlighting, add this to your ``.vimrc``:
+If you want to change the highlighting, add this to your ``.vimrc``: ::
 
-    highlight NoCoverage ctermbg=... guibg=...
-    highlight NoBranchCoverage ctermbg=... guibg=...
-    highlight NoBranchCoverageTarget ctermbg=... guibg=...
+   highlight NoCoverage ctermbg=... guibg=...
+   highlight NoBranchCoverage ctermbg=... guibg=...
+   highlight NoBranchCoverageTarget ctermbg=... guibg=...
```

Comment:
Why the indentation change?

Reply:
I don't know much RST, just went on a doc and it said code block need `::` and 3 whitespaces, might be wrong on that last statement, feel free to revert.

Reply:
It just needs some indentation, doesn't matter how much.  I usually use 2 or 4 spaces.  I found other example blocks in this file using 2 spaces, so I tweaked to match.